src/utils.py

```diff
@@ -166,3 +166,28 @@ def read_view_content(view: Literal["block", "flow", "collection"]) -> Dict[str,
     resp = httpx.get(repo_url + f"/views/{view_filename}")
     resp.raise_for_status()
     return resp.json()
+
+
+def update_block_desc_with_collection_name():
+    existing_block_view = read_view_content("block")
+    for package_name, block_types_dict in existing_block_view.items():
+
+        if "block_types" not in block_types_dict or package_name == "prefect":
+            continue
+
+        for block_type in block_types_dict["block_types"].keys():
+            added_description = (
+                f" This block is part of the {package_name} collection. "
```

Comment:
Adding a new line as well might make this easier to read.
```suggestion
                f"\nThis block is part of the {package_name} collection. "
```

Reply:
I had a new line previously, but in my most recent commit I removed it because I didn't think it would be rendered by the UI

should I put it back?

Reply:
Ah, you're right, the UI doesn't line break on those new lines. Leaving out the newlines is fine then. We can revisit if the UI gets more advanced rendering capabilities.